recipes-core/everest-config/everest-config-migration.bb

```diff
@@ -0,0 +1,28 @@
+DESCRIPTION = "Run the EVerest configuration migration before starting EVerest."
+LICENSE = "CLOSED"
+
+RDEPENDS:${PN} += "everest"
+
+SRC_URI = " \
+    file://01-migrate-configuration.conf \
+    file://everest-migrate-config.sh \
+"
+
+inherit allarch
+
+S = "${WORKDIR}"
+
+FILES:${PN} += " \
+    ${sysconfdir}/systemd/system/everest.service.d/* \
+    ${D}${bindir}/everest-migrate-config.sh \
+"
+
+do_install() {
+    # Systemd configuration 
+    install -d ${D}${sysconfdir}/systemd/system/everest.service.d/
```

Comment:
So far I always used the /etc for systemd configurations because it is documented by systemd that way and also `systemctl edit` places the configs there. Since we are shipping the configuration and it is not a host dependent change, I think you are right. I tested it placing it in `/usr` and it worked too. knowledge++